casadm/cas_main.c

```diff
@@ -632,6 +639,8 @@ static struct cas_param cas_cache_params[] = {
 #define SEQ_CUT_OFF_THRESHOLD_DESC "Sequential cutoff activation threshold [KiB]"
 #define SEQ_CUT_OFF_POLICY_DESC "Sequential cutoff policy. " \
 	"Available policies: {always|full|never}"
+#define SEQ_CUT_OFF_PROMO_THRESHOLD_DESC "Sequential cutoff stream promotion threshold [KiB]"
+#define SEQ_CUT_OFF_PROMO_COUNT_DESC "Sequential cutoff stream promotion request count"
```

Comment:
I suggest to change as per line 568 and line 572 comments

Reply:
Done.